src/diffusers/pipelines/pag/pipeline_pag_pixart_sigma.py

```diff
@@ -132,7 +132,7 @@ def retrieve_timesteps(
     return timesteps, num_inference_steps
 
 
-class PixArtSigmaPAGPipeline(DiffusionPipeline, PixArtPAGMixin):
+class PixArtSigmaPAGPipeline(DiffusionPipeline, PAGMixin):
```

Comment:
We need to update the example doc too and the default value here:

https://github.com/huggingface/diffusers/blob/ebf3ab1477dd480df1b8dd5d97a7b4aa3822716b/src/diffusers/pipelines/pag/pipeline_pag_pixart_sigma.py#L64

https://github.com/huggingface/diffusers/blob/ebf3ab1477dd480df1b8dd5d97a7b4aa3822716b/src/diffusers/pipelines/pag/pipeline_pag_pixart_sigma.py#L167